pii-manager/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
the repo already contains the same license, no need to duplicate

Reply:
Right, since it came from an independent repo, I added a default license when I created it, but now it can use the top-level license.

> Nice work, well designed and like the effort to unit tests. Still some efforts needed to gather more tasks for all languages and ensure all the process is ok with non-western lanagues, but a good progressin structure. One question though: this is a duplicate of your repo, why not using it as an external library?

The idea is indeed for it to be a library, so that code using it just adds it as a dependency (which allows us to version in) and pip-install it. But I got the impression that @ontocord  wanted to merge all tools into this repository, hence the PR. 

Even if inside this repository, it would still be a separate folder that generates an independent package.